src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj

```diff
@@ -16,11 +16,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.UI" Version="1.0.0.21" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.9.0.3" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.6.2.1" />
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.2" />
+    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.2.1.2" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.UI" Version="1.0.0.22" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.10.0.1" />
```

Comment:
Sometimes this stuff can cause problem if updated in my experience with the Xamarin material packages etc.

Reply:
So should we fix the versions? and tell dependabot to ignore them.

Reply:
Yeah maybe, we aren't supporting Xamarin beyond middle of next year anyway with Microsoft stopping support anyway.

Reply:
Reverted